dev/docs/analytics-footer.html

```diff
@@ -0,0 +1,4 @@
+<!-- Google Tag Manager (noscript) -->
```

Comment:
@jayoung-lee can you verify this looks right

Reply:
This code is supposed to appear immediately after the opening <body> tag. Is this file (`analytics-footer.html`) for the body? 

The code itself looks correct!

Reply:
So this will put it near the end of the body. However, looking at this, this is the fallback code specifically for environments where there is no JavaScript enabled, which seems like an edge case.

To truly inject this at the beginning of the body would likely require more engineering work, and providing a new flag upstream in the dartdoc tool, rolling that to the framework, and then using it here. What do you think?